Needs further investigation to see if this can be removed.
The ninstall... function is not callable for some reason, so we need to copy
the definition when using install_cmd.

--- comm-esr31/mozilla/config/rules.mk.orig	2015-06-04 17:42:51.342231683 -0700
+++ comm-esr31/mozilla/config/rules.mk	2015-06-04 17:42:51.344729049 -0700
@@ -1429,7 +1429,7 @@
 nsinstall_is_usable = $(if $(wildcard $(DIST)/bin/nsinstall$(HOST_BIN_SUFFIX)),yes)
 
 define install_cmd_override
-$(1): install_cmd = $$(if $$(nsinstall_is_usable),$$(INSTALL),$$(NSINSTALL_PY) -t) $$(1)
+$(1): install_cmd = $$(if $(wildcard $(DIST)/bin/nsinstall$(HOST_BIN_SUFFIX)),$$(INSTALL),$$(NSINSTALL_PY) -t) $$(1)
 endef
 endif
 endif
